### ISSUE TITLE
Downloading files does not work

### ISSUE THREAD

**Issue by sschuhmann** [flathub/us.zoom.Zoom#209]
System: Fedora 32
DE: Gnome - Wayland

I'm unable to download files from the chat window. A folder for the meeting was created, but contains no files. It would also be nice to have the native filedialog (if possible) and preset the name of the file

**Comment by TheEvilSkeleton** [comment]
Are you using a non-English locale? If so, this might be related to https://github.com/flathub/us.zoom.Zoom/issues/209#issue-707351987.

**Comment by sschuhmann** [comment]
No, I'm using English locale. It is also able to create a folder for the chat in my 'Documents' directory, but is not able to suave the file there.

**Comment by danilkhromov** [comment]
Im having similar issue.
When im trying to download files its only creates folder for current meeting but the folder is empty.

**Comment by tunix** [comment]
Hi,

I have a similar issue. I click on Download inside the chat and nothing happens. I don't see any files being saved. I use a non-English locale but I'm skeptible that it's related to my problem since I don't even see a save dialog.

**Comment by kurono1984** [comment]
The same thing happens to me—it creates virtual folders that, when you download the file from the chat, you can't access from anywhere. I can't believe this bug has been around since 2020 and no one has fixed it yet.
